var/spack/repos/builtin/packages/resolve/package.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Resolve(CMakePackage, CudaPackage):
+    """FIXME: Put a proper description of your package here."""
+
+    homepage = "https://code.ornl.gov/ecpcitest/exasgd/resolve"
+    url = "https://code.ornl.gov/ecpcitest/exasgd/resolve.git"
+    maintainers("cameronrutherford", "pelesh", "ryandanehy")
+
+    # version("0.1.0", submodules=False, branch=develop)
+    # version("develop", submodules=False, branch=develop)
+
+    variant("klu", default=True, description="Use KLU, AMD and COLAMD Libraries from SuiteSparse")
+
+    depends_on("suite-sparse", when="+klu")
+
+    # # suite-sparse is not a CudaPackage so we cannot specify cuda_arch
+    # # for other dependencies we could
```

Comment:
Some SuiteSparse packages do have CUDA backends, but we don't use them. I sligtly eddited this comment to reflect that:
```suggestion
    # # We do not use CUDA modules of suite-sparse so we don't specify cuda_arch
    # # For other dependencies we could
```